README.md

```diff
@@ -191,6 +193,26 @@ var config = {
 var authClient = new OktaAuth(config);
 ```
 
+##### PKCE Oauth flow
+
+By default the `implicit` Oauth flow will be used. It is widely supported by most browsers. PKCE is a newer flow which is more secure, but does require certain capabilities from the browser. For strongest security with the widest browser support, it is recommended to test for and use PKCE if supported or fall back to `implicit` if not.
```

Comment:
I think we should avoid to suggest to fall back to `implicit`, since in this paragraph we are trying to show the strongest security option.

Reply:
I have updated README to no longer recommend fallback